examples/examples/z_get.rs

```diff
@@ -24,19 +24,18 @@ async fn main() {
     let (config, selector, value, target, timeout) = parse_args();
 
     println!("Opening session...");
-    let session = zenoh::open(config).res().await.unwrap();
+    let session = zenoh::open(config).await.unwrap();
 
     println!("Sending Query '{selector}'...");
     let replies = session
         .get(&selector)
-        // // By default get receives replies from a FIFO. 
-        // // Uncomment this line to use a ring channel instead. 
+        // // By default get receives replies from a FIFO.
+        // // Uncomment this line to use a ring channel instead.
         // // More information on the ring channel are available in the z_pull example.
         .with(zenoh::handlers::RingChannel::default())
```

Comment:
I think it slipped in from some other PR but this line should be commented out.

```suggestion
        // .with(zenoh::handlers::RingChannel::default())
```

Reply:
I've found the culprit 😛 https://github.com/eclipse-zenoh/zenoh/pull/903/files#diff-de144b4bbbdf20af6b0b2ed3a9fc9301218263abb726d71700859c1c270caeecR36